plugin/src/main/java/com/denizenscript/denizen/objects/InventoryTag.java

```diff
@@ -2356,6 +2356,28 @@ else if ((object.inventory instanceof FurnaceInventory)) {
             }
             return list;
         });
+
+        // <--[tag]
+        // @attribute <InventoryTag.find_empty_slots>
+        // @returns ListTag(Number)
+        // @description
+        // Returns the index of all the empty slots in an inventory.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "find_empty_slots", (attribute, object) -> {
+            if (object == null) {
+                return new ListTag();
+            }
+            ListTag indexes = new ListTag(object.getContents().length);
+            // Denizen starts at 1
+            int index = 1;
+            for (ItemStack itemStack : object.getContents()) {
+                if (itemStack == null) {

```

Comment:
should `|| type == air`, I don't think there's any guarantee that air items _have to become_ null.